ArcadiaParties.API/Startup.cs

```diff
@@ -36,7 +38,9 @@ public void ConfigureServices(IServiceCollection services)
                 c.SwaggerDoc("v1", new OpenApiInfo { Title = "ArcadiaParties.API", Version = "v1" });
             });
 
-            services.AddMediatR(typeof(Temp));
+            services.AddMediatR(typeof(Temp).Assembly);
```

Comment:
By the way, you can remove `Temp` class and use any of real classes to get an assembly.

Reply:
Temp class has removed, changed MediatR DI